docs/customising/make_a_new_theme.md

```diff
@@ -96,6 +105,32 @@ right place within your Alaveteli installation.
 When it's finished, you can see that your theme (that is, all the files from
 the repo) has been reproduced in `lib/themes/`.
 
+<div class="attention-box helpful-hint">
+  <p>
+    If you're just following this guide to see what is possible and have no
+    interest in saving the changes you're about to make, you can skip this bit!
+  </p>
+  <p>
+    On the command line (back on your own machine rather than inside Vagrant
+    box), `cd` to your theme's folder inside `lib/themes/`. This is where your
+    theme code will live, and is where you should issue your git commands from
+    to commit and push your changes.
+  </p>
+  <p>
+    You now need to issue this command:
+
+    <pre><code>git checkout master</code></pre>
+
+    This is needed because, unless you've told it explicitly to do otherwise,
+    the rake task won't checkout a branch (instead, it leaves you in a 
+    "detached HEAD" state) -- see the [note about branches](#branches-within-your-theme-repo)
```

Comment:
This doesn't render as a link for some reason:

![screen shot 2016-04-01 at 14 47 37](https://cloud.githubusercontent.com/assets/282788/14208725/b5596a3e-f818-11e5-8c39-218bb1d01e7e.png)